core/new-gui/src/app/dashboard/component/feature-container/user-project-section/project-page/ngbd-modal-remove-project-file/ngbd-modal-remove-project-file.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { NgbActiveModal } from "@ng-bootstrap/ng-bootstrap";
+import { Observable, forkJoin } from "rxjs";
+import { UserProjectService } from "../../../../../service/user-project/user-project.service";
+import { DashboardUserFileEntry } from "../../../../../type/dashboard-user-file-entry";
+import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
+import { UserFileService } from "../../../../../service/user-file/user-file.service";
+
+@UntilDestroy()
+@Component({
+  selector: "texera-remove-project-file-modal",
+  templateUrl: "./ngbd-modal-remove-project-file.component.html",
+  styleUrls: ["./ngbd-modal-remove-project-file.component.scss"]
+})
+export class NgbdModalRemoveProjectFileComponent implements OnInit {
+  // @Input() addedFiles!: DashboardUserFileEntry[];
+  @Input() addedFiles!: ReadonlyArray<DashboardUserFileEntry>;
+  @Input() projectId!: number;
```

Comment:
clean up and please remove local cache.

Reply:
discussed and resolved issue of local cache